plugin/evm/vm.go

```diff
@@ -351,6 +362,10 @@ func (vm *VM) Initialize(
 	vm.chain = ethChain
 	lastAccepted := vm.chain.LastAcceptedBlock()
```

Comment:
Let's log this out (but correctly unlike coreth): https://github.com/ava-labs/coreth-internal/blob/9fb7a2025a20f3a5546683571f1066cae932457c/plugin/evm/vm.go#L462